nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/xml/TestWriteXMLResult.java

```diff
@@ -85,7 +85,7 @@ public void testRecordNameIsNullSchemaIdentifierMissing() {
 
         try {
             new WriteXMLResult(recordSet.getSchema(), new SchemaNameAsAttribute(),
-                    out, true, ALWAYS_SUPPRESS, NO_WRAPPING, null, "root", null, "UTF-8", DATE_FORMAT, TIME_FORMAT, TIMESTAMP_FORMAT);
+                    out, true, false, ALWAYS_SUPPRESS, NO_WRAPPING, null, "root", null, "UTF-8", DATE_FORMAT, TIME_FORMAT, TIMESTAMP_FORMAT);

```

Comment:
I didn't see a unit test where omitDeclaration is set to true, let's make sure there is one and we verify that the header is not written out when true.

Reply:
Sure @mattyb149 I will add it. Thanks